<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DRCwind</class>
 <widget class="QMainWindow" name="DRCwind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CUDA-Based Design Rule Checker</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionRunDRC"/>
  </widget>
  <widget class="ErrorList" name="errorList">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionRunDRC">
   <property name="text">
    <string>Run DRC</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ErrorList</class>
   <extends>QDockWidget</extends>
   <header>widgets/ErrorList.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
